### PR TITLE
Update debugging-workflows.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/framework/windows-workflow-foundation/debugging-workflows.md [hunk 1]
```diff
@@ -4,25 +4,32 @@ ms.date: "03/30/2017"
...
+
+## Debugging in XAML
+
+If a workflow has paused at a breakpoint in the designer, the workflow can also be debugged in XAML. To view the point of execution in XAML, select **XAML View** in the workflow designer when workflow execution is paused. Debugging can be switched back to the designer by re-opening the workflow in the designer from the solution explorer. For more information, see [How to: Debug XAML with the Workflow Designer](/visualstudio/workflow-designer/how-to-debug-xaml-with-the-workflow-designer).
+
+## Debugging in Code
+
+Code breakpoints can be used in [!INCLUDE[netfx_current_short](../../../includes/netfx-current-short-md.md)] in the same way that they can be used in other imperative applications. Click the left margin of the code pane to create a code breakpoint, or press <kbd>F9</kbd> to place a breakpoint at the cursor location.
```
Comment (rpetrusha) [inline docs/framework/windows-workflow-foundation/debugging-workflows.md]:
```suggestion
To set a breakpoint, click the left margin of the code pane to create a code breakpoint, or press <kbd>F9</kbd> to place a breakpoint at the cursor location.
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for making these changes, @Youssef1313. I've suggested one additional change.

File: docs/framework/windows-workflow-foundation/debugging-workflows.md (hunk 1 above)
Comment (Youssef1313) [reply docs/framework/windows-workflow-foundation/debugging-workflows.md]: @rpetrusha for being late, just noticed this now.

I think the repetition of `set a breakpoint` and `create a breakpoint` is unintentional. so I'm modifying the suggestion to:

```suggestion
To set a breakpoint, click the left margin of the code pane, or press <kbd>F9</kbd> with the cursor at the line where you want to set it.
```

**Comment by rpetrusha** [comment]
Thanks, @Youssef1313. I'll merge your PR now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**